### DISCUSSION TITLE
Feedback v1.5.3; 04.11.2022

### DISCUSSION THREAD

**Discussion by tlaszlozoltan** [fedorovvl/tso_client#121] [General]
With this client, the game freeze. When I jump ferquently between mission and Island. For example I did some (10+) Riches of the Mountain scenario, and need 3 times too restart client, but this happens too when I go 2 mission own and as guest and jump between adventures.

**Comment by fedorovvl** [comment]
and u don't facing it on 1.5.2 or 1.5.1?
what you mean freezing?
give more info pls

**Reply by tlaszlozoltan** [reply]
> and u don't facing it on 1.5.2 or 1.5.1?

Yes, it is only in 1.5.3. We in our guild use your amazing client ty vm! :) And tested others too. Some of us went back to 1.5.2 and work correctly.

**Reply by tlaszlozoltan** [reply]
> and u don't facing it on 1.5.2 or 1.5.1? what you mean freezing? give more info pls

Freeze: hourgass, windows shows white screen: not response. When we go back to Island. Maybe some script or Idk work hard. When we click on game, windows ask what we would like to do. Close program or w8 for response. After a 30 sec I usually close and restart the game.

**Reply by fedorovvl** [reply]
how can i reproduce?

**Reply by BasilMas** [reply]
I think the problem is more likely to be related to the ongoing event. More players are online, which means more requests for game servers.  
I faced up with freezing and crashing before, in other events and with another client version.
Upd
I remember the problem first started on Valentine's-22 Event, when I had to restart the client after 5-6 Lake Treasures.

**Reply by tlaszlozoltan** [reply]
> how can i reproduce?

Do some Riches of the Mountain scenario. I use simple prov house with buff, very fast 3-5 minute / scenario. 3 jump and 3 jump back / scenario.

**Comment by igorSof** [comment]
Зависания есть в игре в основном от частых переходов с острова на остров или в приключение. такая проблема возникает чаще на 32 версии чем на 64. А вот 64 сейчас нет, только 32 и реально 5-6 приключений и надо перезапустить игру иначе всё равно вылетит. Верните версию х64

**Reply by BasilMas** [reply]
я играю только на х64, зависания при переходах никуда не делись, в ивент они усугубились

**Reply by igorSof** [reply]
А где взяли версию клиента х64? ведь сейчас только х32 доступна.

**Reply by fedorovvl** [reply]
текущая версия под обе платформы.. в Adobe AIR\Versions\1.0\Resources\ 2 версии плагина NPSWF32.dll и NPSWF64.dll
система сама выбирает какой использовать

**Reply by igorSof** [reply]
после смены фазы ивента зависания и вылеты исчезли, стало как раньше нормально.

**Reply by fedorovvl** [reply]
чудеса да и только)

**Reply by BasilMas** [reply]
> после смены фазы ивента зависания и вылеты исчезли, стало как раньше нормально.

может после 1й активной фазы, когда толпы народу в толпы приклов ходили, то  и нагрузка на игровые сервера была выше, не?
или прыжки с домашнего острова в прикл и обратно для 10-15-20 приключений за раз вызывали рефреш зоны, а при рефреше утекает память, другими словами частые рефреши приводят к проблемам с соединением?
Еще более другими словами может быть дело не в клиенте/флешке/версии Air, а в нагрузке на игровые сервера? Активная фаза ивента ее увеличивает, вялое меживентье - почти никаких проблем? О чем я и писал ранее

**Reply by igorSof** [reply]
Еще немного собрал статистики. Да, если не играть, а просто оставить клиента, то ничего не виснет, а вот если активно играть, то полчаса и вылет как в ивент, хотя сейчас опять мини ивент включили.

**Reply by fedorovvl** [reply]
все указывает на утечку памяти где то в клиенте.. но определить её пока не представляется возможным

**Reply by BasilMas** [reply]
> все указывает на утечку памяти где то в клиенте.. но определить её пока не представляется возможным

Так оно и раньше указывало - проблема скорее всего связана с рефрешем, не зря же он был отключен.
Ручной рефреш по Ф2 примерно раз на 10-15-28 приведет к проблемам с клиентом, даже если никуда не ходить (приклы/друзья). Смена зоны - друзья/остров приключения/домашний остров вызывает тот же рефреш автоматически, т.е. чем больше таких переходов за сеанс - тем быстрее игра зависнет.

**Comment by igorSof** [comment]
Еще в последней версии 1.5.3 появилась проблема, что не выпадают коллекции через 8 часов, если игра просто открыта новый выпадений не будет пока не обновлю зону, реально сервер коллекции высыпает, но они не отображаются, друзья видят на моем острове коллекции а я нет, обновлю зону и они появляются.

**Reply by fedorovvl** [reply]
флешка одна и та же на любой версии клиента.. вы можете с уверенностью сказать что на 1.5.2 или 1.5.1 все работает как надо?

**Reply by igorSof** [reply]
нет, зависать стало часто как только убрали версию x64

**Reply by igorSof** [reply]
вылетает (просто закрывается программа) в основном когда используется скрипты, но вот зависает (белый прозрачный экран) может и просто в любой момент произойти.
Повторюсь, на x64 версии такое тоже было, но гораздо реже, на 32 версии я за день раз 10 запускаю клиента.

**Reply by fedorovvl** [reply]
возможно причина в смене версии Adobe Air.. в 1.5.1 перешли на 32.0.0.125

**Reply by igorSof** [reply]
возможно из-за смены Adobe Air.
Друзья тоже пишут что хуже стало.

**Reply by igorSof** [reply]
а вот коллекции не проявляются сами, надо принудительно обновлять зону. держу клиент всегда открытым только из-за этого чтобы максимум их собрать

**Reply by MrBadger** [reply]
I have also realised that new collectibles  do not appear without a refresh.  Friends have told me they marked them when I thought I had found all.

Can't say for sure if it's every time or just sometimes.

**Comment by FernandoTLeite** [comment]
I can't open the client. The following message appears:

Message: Exception of type 'System.OutOfMemoryException' was thrown.
Source: AutoUpdater.NET
Target: Void Start(System.String, System.Reflection.Assembly)
